### PR TITLE
bench improvements

### PR DESCRIPTION
**Pull request by ronag**

### CODE REVIEW

**Comment by ronag** [comment]
@ShogunPanda

File: README.md
Comment (mcollina) [inline README.md:45]: This looks odd: how come 50 connections have 1/5 op/sec than 1 connection?

There is likely something wrong in our benchmark.

Comment (ronag) [reply README.md:45]: It’s how the bench are built. It’s counting “ops” however 50 connections do more per ops in order to fully saturate the connections.

Comment (ronag) [reply README.md:45]: I think I can probably fix that

Comment (ShogunPanda) [reply README.md:45]: Confirmed. Those are not requests but ops so indeed is misleading. 

@ronag You can probably copy the table production code from cronometro and work with raw results to apply the multiplication that was present in the old version of the benchmarks.

Or we can merge this and I'll take care of that. Let me know.

**Comment by ShogunPanda** [comment]
LGTM!

File: README.md
Comment (ronag) [reply README.md]: I just made sure that the number of parallel requests are the same and then it's ok.

**Comment by ronag** [comment]
@mcollina PTAL

**Comment by mcollina** [comment]
the benchmark workflow in CI stalled and blocked all other PRs. There is something not working right now on this PR.

**Comment by ronag** [comment]
@ShogunPanda do you think you could make us a custom printer for cronometro that prints it in a way we can just copy past into README? In particular, multiply the results by `parallelRequests` and printing `reqs/s` instead of `ops/s`?

**Comment by ShogunPanda** [comment]
Sure, I definitely can. Once you land this (so benchmark code is fixed) I'll work on changing the printer.

**Comment by ronag** [comment]
I'm still a little confused by the results:

The difference with slowest is significantly larger for the single socket case. Also in the multi socket case `request` is faster than `stream`.

However, I think this is as a good as I can get it at the moment.

**Comment by ronag** [comment]
> Sure, I definitely can. Once you land this (so benchmark code is fixed) I'll work on changing the printer.

Excellent. I'll merge once Matteo approves.

**Review by dnlup — APPROVED** [review]

**Comment by mcollina** [comment]
ci is failing :/

**Comment by ronag** [comment]
> ci is failing :/

GitHub actions seems to be having problems.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**
